baselines/flanders/flanders/conf/base.yaml

```diff
@@ -0,0 +1,52 @@
+---
+# this is the config that will be loaded as default by main.py
+# Please follow the provided structure (this will ensuring all baseline follow
+# a similar configuration structure and hence be easy to customise)
+#defaults:
+#  - _self_
+#  - override hydra/hydra_logging: disabled
+#  - override hydra/job_logging: disabled
+
+hydra:
+  output_subdir: null
+  run:
+    dir: .
```

Comment:
it's preferable if outputs go to the directory automatically generated by hydra (that's how all baselines do it). You are welcome to rename that directory (defaults to `outputs/`) if you don't like it.

Reply:
Now I save it here `save_path = HydraConfig.get().runtime.output_dir`, however I don't use the history that the strategy outputs as FedProx does; I still delegate this to the strategy by means of the evaluation function. Do you think it is better to use the history?

Reply:
I think this is fine as long as all outputs that need to do plotting later are stored. I see you generate a `outputs/all_results.csv`. I guess this is sufficient?